odo-extension-entrypoint.sh

```diff
@@ -14,55 +14,32 @@
 ODO_EXTENSION_DIR=/codewind-workspace/.extensions/codewind-odo-extension
 
 ROOT=$1
-LOCAL_WORKSPACE=$2
-PROJECT_ID=$3
-COMMAND=$4
-CONTAINER_NAME=$5
-AUTO_BUILD_ENABLED=$6
-LOGNAME=$7
-START_MODE=$8
-DEBUG_PORT=$9
-FORCE_ACTION=${10}
-FOLDER_NAME=${11}
-DEPLOYMENT_REGISTRY=${12}
-COMPONENT_TYPE=${13}
-
-####################
-# Hardcode parameters value for testing purpose
-ROOT=/codewind-workspace/nodejs-ex
-PROJECT_ID=000
-FOLDER_NAME=odo-log
-COMPONENT_TYPE=nodejs
-
-####################
+PROJECT_ID=$2
+COMMAND=$3
+COMPONENT_TYPE=$4
+COMPONENT_NAME=$5
+FOLDER_NAME=$6
+AUTO_BUILD_ENABLED=$7
```

Comment:
You have removed the deployment registry. Don't we need that when pushing images?

Reply:
odo doesn't need deployment registry, odo will use s2i image and create build-time & run-time containers for us, odo can build and deploy the application source code directly to the OpenShift cluster (the cluster that we already login) by using the build-time and run-time containers. It's different from other project types, for other projects we have to build the application image by ourselves then push it to the deployment registry, then use helm to deploy it to the Kube cluster.